microsoft/testsuites/security/fips.py

```diff
@@ -18,10 +27,106 @@
     Tests the functionality of FIPS enable
     """,
 )
-class Fips(TestSuite):
-    _expected_failure_pattern = re.compile(
-        "Error setting digest\r\n.*EVP_DigestInit_ex:disabled for FIPS.*", re.M
+class FipsTests(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Ensures that an AZL machine is fips enabled.
+        """,
+        priority=1,
+        requirement=simple_requirement(
+            supported_os=[CBLMariner],
+        ),
+    )
+    def verify_fips_is_enabled_azl(
```

Comment:
For test case names, how about like `verify_azl_fips_is_enabled`? Move azl to the front, so it's easier to identify azl cases.